index.html

```diff
@@ -3,24 +3,25 @@
     <head>
         <meta charset="utf-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <title>JW Player example</title>
-        <meta name="description" content="JW Player example" />
-        <meta name="author" content="Vualto" />
+        <title>JW Player Example</title>
+        <meta name="description" content="JW Player Example" />
+        <meta name="author" content="JW Player" />
         <style>
-            #vuplay-container {
+            #studiodrm-container {
                 width: 50%;
             }
         </style>
     </head>
     <body>
-        <div id="vuplay-container"></div>
+        <div id="studiodrm-container"></div>
+        <!--TODO: For the JW Player src - look at using the version hosted on JW Player's CDN-->
```

Comment:
Can we stick to writing the TODO's one way. I prefer this over the shouty one as iirc this is how we do it everywhere else.

Reply:
Yes. It was a moment of pure madness!